src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -381,23 +422,20 @@ export async function extractTiffLocations(
           return null;
         }
 
-        const centerX = (bbox[0] + bbox[2]) / 2;
-        const centerY = (bbox[1] + bbox[3]) / 2;
-        // bbox is not epsg:2193
         const targetProjection = Projection.get(2193);
         const sourceProjection = Projection.get(sourceEpsg);
 
-        const [x, y] = targetProjection.fromWgs84(sourceProjection.toWgs84([centerX, centerY]));
-        if (x == null || y == null) {
+        const [ulX, lrY, lrX, ulY] = reprojectIfNeeded(bbox, sourceProjection, targetProjection);
```

Comment:
we could stop destructuring this and leave it as a bbox (or eventually a polygon :) )

Reply:
fixed